pkg/kldexerciser/kldworker.go

```diff
@@ -383,25 +372,31 @@ func (w *Worker) Run() {
 			if err != nil {
 				w.error("TX send failed (%d/%d): %s", i, w.Exerciser.TxnsPerLoop, err)
 			} else {
-				w.Nonce++
 				txHashes = append(txHashes, txHash)
 			}
+			w.Nonce++
```

Comment:
I'm worried this is going to break the nonce logic.
I agree in the _special_ case of a failure with `status=1`, we need to bump the Nonce on a failure, but in the general case where transactions cannot be submitted if we bump the nonce we'll create gaps.

Reply:
I will move this one back inside the `if else` block. I agree that the issue I was trying to address was a very specific case that does not need to be addressed if it risks the logic for the broader, more likely scenarios.